### PR TITLE
Rename dword_518068 => gAiInitialized and change type to bool.

### PR DESCRIPTION
**Pull request by brcolow**
I am not sure if this is useful at all, but wanted to try my hand at making the world's smallest contribution.

Let me know if I did anything wrong. Never done this type of work before.

Cheers.

### CODE REVIEW

**Comment by alexbatalov** [comment]
Change declaration in `combat_ai.h`, replace `0` with `false`, `1` with `true` whenever this value is used.

**Comment by brcolow** [comment]
Ah yes. Of course. This probably took more work for you to review than do it yourself, so thank you for graciously holding my hand for this contribution! :)

Should be good to go now (did a fixup commit).

File: src/combat_ai.c [hunk 1]
```diff
@@ -39,7 +39,7 @@ int gAiPacketsLength = 0;
 AiPacket* gAiPackets = NULL;
 
 // 0x518068
-int dword_518068 = 0;
+bool gAiInitialized = 0;
```
Comment (alexbatalov) [inline src/combat_ai.c]: Forgot `false` here

Comment (brcolow) [reply src/combat_ai.c]: Fixed.

**Pull request merged**